### PR TITLE
plocate plugin

### PR DESCRIPTION
**Pull request by pielgrzym**
Hi,

Thanks for this great tool! I created a simple plugin that uses locate/plocate command to search for file/path name and pipes the result to fzf. User can select desired path and NNN will cd into given path. Plugin has fallback in case user uses locate and detects if path is a file or directory.

### CODE REVIEW

File: plugins/plocate [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+printf "find: "
+read -r pattern
+
+if type plocate >/dev/null 2>&1; then
+    dest_path=$(plocate "$pattern" | fzf)
+else
+    dest_path=$(locate "$pattern" | fzf)
+fi
+
+if [[ -f "$dest_path" ]]; then
```
Comment (N-R-K) [inline plugins/plocate:23]: `[[ ]]` is a bash feature. Change it to `[ ]` if you want it to be posix compliant.

File: plugins/plocate [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Shell: POSIX compliant
+# Author: Jakub Nawalaniec
+
+. "$(dirname "$0")"/.nnn-plugin-helper
+
+printf "find: "
+read -r pattern
+
+if type plocate >/dev/null 2>&1; then
+    dest_path=$(plocate "$pattern" | fzf)
```
Comment (N-R-K) [inline plugins/plocate:18]: I'd avoid depending on fzf. `nnn` has a listing feature alreay, see [quick grep](https://github.com/jarun/nnn/tree/master/plugins#quick-grep-using-rg) example (although I'm wondering why the example is using `eval`).

**Review by N-R-K — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
Yes, we have too many plugins already to find, cd etc.

File: plugins/plocate (hunk 1 above)
Comment (pielgrzym) [reply plugins/plocate]: Done :heavy_check_mark:

File: plugins/plocate (hunk 2 above)
Comment (pielgrzym) [reply plugins/plocate]: I tried the code above but the result is a root level directory list not a list of all paths I can easily filter to see what has been matched. I passed "+l" as in grep plugin. Also it seems length of results is heavily limited this way. When I search for a very common dir name (like: "git") I get an error "file limit exceeded" (unlike fzf, which just flys with whatever I toss at it). Any way around this?

**Comment by pielgrzym** [comment]
Ah, I see - no problem, shall I close the PR? :)

**Comment by jarun** [comment]
Yes.

**Comment by jarun** [comment]
I closed it.

**Comment by jarun** [comment]
You may want to try the fzcd and fzopen plugins.

**Comment by pielgrzym** [comment]
Thanks for tips - those plugins just search a subpath, not mlocate/slocate/plocate file db :)

**Pull request closed without merging**